must-gather/collection-scripts/pre-install.sh

```diff
@@ -43,13 +48,15 @@ deploy(){
 }
 
 labels(){
+    if [ -n "${storageClusterPresent}" ]; then
      oc label pod -n openshift-storage "${HOSTNAME}"-helper must-gather-helper-pod=''
+    fi
 }
 
 check_for_debug_pod(){
     debug_pod_name=$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $1}')
     # sleep for 60 seconds giving time for debug pod to get created
-    sleep 60 
+    sleep 60
```

Comment:
It is good to have a clean code. I have one request can raise another pr fixing all the spaces in one commit.
Every-time Having separate commit for spaces is clumsy. 
Can we have a test for this so that spaces cannot be added in the first place.
for go code we have golangci check for this.

Reply:
Sure, we can get this merged and from the next time, we can use your fix for fixing empty spaces.